src/components/Match/MatchHeader.tsx

```diff
@@ -36,10 +32,12 @@ const MatchHeader = ( { topSuggestion, hideObservationStatus }: Props ) => {
 
   const observationStatus = ( ) => {
     let confidenceType = "may_have_observed";
-    if ( confidence >= 93 ) {
-      confidenceType = "observed";
-    } else if ( confidence >= 50 && confidence < 93 ) {
-      confidenceType = "likely_observed";
+    if ( confidence ) {
+      if ( confidence >= 93 ) {
+        confidenceType = "observed";
+      } else if ( confidence >= 50 && confidence < 93 ) {
+        confidenceType = "likely_observed";
+      }
```

Comment:
Do you have an idea what drives `93` and `50`? If it's known, maybe we could move these to descriptively named file-level constants?

Reply:
The numbers are from here (but the text in Figma is outdated it seems)
https://www.figma.com/design/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?node-id=23127-92653&t=FKfMR44IGsHayrfF-4

Reply:
awesome, thanks y'all! Totally makes sense to name them after the confidenceType enums they're driving